contextproject-ms3/src/main/java/contextproject/controllers/CLIController.java

```diff
@@ -0,0 +1,97 @@
+package contextproject.controllers;
+
+import contextproject.loaders.FolderLoader;
+import contextproject.models.Playlist;
+
+import java.io.IOException;
+import java.util.Scanner;
+
+/**
+ * Controls CLI for time being.
+ * 
+ * @author Felix
+ */
+public class CLIController {
+
+  private String fileName;
+  private Playlist result;
+
+  /**
+   * Input is initialised to empty string and result to null.
+   */
+  public CLIController() {
+    fileName = "";
+    result = null;
+
+  }
+  /**
+   * This will start a simple Command Line Interface for our first working version. Optionally to be
+   * included in later versions.
+   */
+  public void run() {
+    int input = 0;
+    Scanner sc = new Scanner(System.in);
+    System.out.println("Welcome to our application!");
+    while (input != 4) {
+      System.out.println("Option 1: Construct playlist! Enter foldername");
+      System.out.println("Option 2: About");
+      System.out.println("Option 3: Play music!");
```

Comment:
I thought we didn't include this in the current sprint, so we should remove it for now. (else the user may be confused when this option does nothing)